docs/docs/glossary.md

```diff
@@ -70,11 +83,11 @@ Content and data's origin point, typically integrated into Gatsby with [source p
 
 ### Database
 
-A database is a structured collection of data or content. Often a [CMS](#cms) will save to a database using [back-end technologies](#back-end). They're often accessed in Gatsby via a [source plugin](#source-plugin)
+A database is a structured collection of data or content. Often a [CMS](#cms) will save to a database using [back-end technologies](#backend). They're often accessed in Gatsby via a [source plugin](#source-plugin)
```

Comment:
```suggestion
A database is a structured collection of data or content. Often a [CMS](#cms) will save to a database using [backend technologies](#backend). They're often accessed in Gatsby via a [source plugin](#source-plugin)
```

Should the hyphen be omitted here too? It looks like it was changed in a couple other places.

Reply:
Yes, you got it! I thought I found them all with find and replace, but obviously not 😅